src/Avatar/__tests__/StyledAvatar.test.js

```diff
@@ -19,7 +20,7 @@ describe('<StyledAvatar />', () => {
           />
         );
 
-        expect(avatar.find('Flex')).toHaveProp({
+        expect(avatar.find(Flex)).toHaveProp({
```

Comment:
A few cases of `find` by string stopped working, and needed to be switched to `find` by ComponentClass.